pow-migration/src/monitor/mod.rs

```diff
@@ -24,19 +25,20 @@ pub const READY_PERCENTAGE: u8 = 80;
 ///   Sender: Validator address
 ///   Recipient: Burn address
 ///   Value: 1 Luna
-///   Data: TBD
+///   Data: Hash of the generated `GenesisConfig`
 ///
-pub fn generate_ready_tx(validator: String) -> OutgoingTransaction {
+pub fn generate_ready_tx(validator: String, hash: &Blake2bHash) -> OutgoingTransaction {
     info!(
         validator_address = validator,
+        pos_genesis_hash = hash.to_hex(),
```

Comment:
`Blake2bHash` implements `Display`
```suggestion
        pos_genesis_hash = %hash,
```